packages/cozy-procedures/src/components/Overview.jsx

```diff
@@ -1,24 +1,103 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { withRouter } from 'react-router'
-import Topbar from './Topbar'
-import creditApplicationTemplate from '../templates/creditApplicationTemplate'
+import { withClient } from 'cozy-client'
+import { AdministrativeProcedure } from 'cozy-doctypes'
+import CozyRealtime from 'cozy-realtime'
 import {
+  Alerter,
   Title,
   SubTitle,
   Caption,
   Button,
   translate
 } from 'cozy-ui/transpiled/react'
 
+import Topbar from './Topbar'
+import creditApplicationTemplate from '../templates/creditApplicationTemplate'
+
 class Overview extends React.Component {
+  realtime = null
+
+  componentWillUnmount() {
+    if (this.realtime) {
+      this.realtime.unsubscribeAll()
+    }
+  }
+
   navigateTo = view => {
     const { location, router } = this.props
     const rootPath = location.pathname
     const separator = rootPath.endsWith('/') ? '' : '/'
     router.push(`${rootPath}${separator}${view}`)
   }
 
+  ensureRealtime = () => {
+    if (!this.realtime) {
+      this.realtime = new CozyRealtime({ client: this.props.client })
+    }
+  }
+
+  handleRealtime = jsonFile => job => {
+    console.log('realtime: the zip is ready', new Date(), job)
+    Alerter.success('Your procedure is available in your Drive.')
+    this.props.client.collection('io.cozy.files').destroy(jsonFile)
+  }
+
+  submitProcedure = async () => {
+    const { client } = this.props
+    // TODO: id of the Administratif dir
+    const dirId = 'c56777a2ab386f28f6e12c034e0c0162'
```

Comment:
Just added to the template a pathToSave. You can use it with `ensureDirectoryExists` to get the id 

https://github.com/cozy/cozy-libs/pull/540/files#diff-986178431654e5c01cfcb9dabd2331feR254